### PR TITLE
Group owner can self restore the group

### PR DESCRIPTION
**Pull request by Batre-MSFT**
The group owner self can restore deleted Office365 group. Added the steps to indicate same.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 1]
```diff
@@ -20,6 +20,18 @@ description: "Learn how to restore a deleted Office 365 Group using the Exchange
 ---
 
 # Restore a deleted Office 365 Group
+
+The owner of an Office 365 group can restore the group by self.
+
+Steps:
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]:
```suggestion
```

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 2]
```diff
@@ -20,6 +20,18 @@ description: "Learn how to restore a deleted Office 365 Group using the Exchange
...
+1) Access the deleted groups page by logging in to this link: https://outlook.office.com/people/group/deleted 
+or 
+Login to Outlook for Web (OWA), click on Manage groups option, present under the Groups node, and then click on "Deleted" option
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]:
```suggestion
log in to Outlook for Web (OWA), select the **Manage groups** option under the **Groups** node, and then choose **Deleted**.
```

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 3]
```diff
@@ -20,6 +20,18 @@ description: "Learn how to restore a deleted Office 365 Group using the Exchange
...
+
+2) The Office 365 groups deleted by the owner will be displayed on the right hand pane. 
+3) From here, the group owner can click on "Restore" and the group will be restored
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]:
```suggestion
```

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 4]
```diff
@@ -20,6 +20,18 @@ description: "Learn how to restore a deleted Office 365 Group using the Exchange
...
+
+If the deleted group doesn't show here, the owner should contact person with administrative permissions.
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]:
```suggestion
If the deleted group doesn't appear here, the group owner should contact a person with administrative permissions.
```

**Comment by mypil** [comment]
@Batre-MSFT - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by Batre-MSFT** [comment]
@mypil done

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 5]
```diff
@@ -20,6 +20,18 @@ description: "Learn how to restore a deleted Office 365 Group using the Exchange
 ---
 
 # Restore a deleted Office 365 Group
+
+The owner of an Office 365 group can restore the group by self.
```
[the remark this answers is not in the document]
Comment (mypil) [reply Office365-Admin/create-groups/restore-deleted-group.md]: @Batre-MSFT - Please merge this commit as well. Thanks.

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Pull request merged**